app/pages/Help.js

```diff
@@ -1,5 +1,6 @@
+import { th } from 'date-fns/locale';
 import React from 'react';
-import { Image, Accordion, Icon, Table } from 'semantic-ui-react';
+import { Image, Accordion, Icon, Table, AccordionTitle, AccordionContent } from 'semantic-ui-react';
```

Comment:
We do not need to import `AccordionTitle`and `AccordionContent`since we already have access to these via the `Accordion` class. This will require two other changes as well, which I will state further into the review.
```suggestion
import { Image, Accordion, Icon, Table } from 'semantic-ui-react';
```